docs/en/sql-reference/functions/logical-functions.md

```diff
@@ -172,11 +168,9 @@ xor(val1, val2...)
 
 **Returned value**
 
-- `1`, for two values: if one of the values evaluates to `false` and other does not,
-- `0`, for two values: if both values evaluate to `false` or to both `true`,
-- `NULL`, if at least one of the inputs is `NULL`
-
-Type: [UInt8](../../sql-reference/data-types/int-uint.md) or [Nullable](../../sql-reference/data-types/nullable.md)([UInt8](../../sql-reference/data-types/int-uint.md)).
+- `1`, for two values: if one of the values evaluates to `false` and other does not. [UInt8](../../sql-reference/data-types/int-uint.md) or [Nullable](../../sql-reference/data-types/nullable.md)([UInt8](../../sql-reference/data-types/int-uint.md)).
```

Comment:
Here, including the return types 3 times looks silly. Can we keep the previous formatting?


(same applies to the other changed places in this file)